genome/pyrule/envs/tree.post-deploy.sh

```
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+cd $CONDA_PREFIX/lib
+    git clone https://github.com/motu-tool/fetchMGs.pl.git
+    git checkout v1.2
+
+    echo `realpath fetchMGs.pl/fetchMGs.pl` '$@' > ../bin/fetchMGs
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)